src/content/docs/how-to-open-a-pull-request.mdx

```diff
@@ -14,7 +14,7 @@ Some examples of this are:
 
 1. Do not edit files directly through GitHub – while you can, it's not a good idea.
 2. Make sure the PR title follows [our convention](#prepare-a-good-pr-title).
-3. Make sure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
+3. When submitting a PR, put an x in each of the boxes like this: `[x]`. Make sure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
```

Comment:
I don't like this, you should put focus on completing the checklist, not on putting the X there

```suggestion
3. Follow the PR checklist and tick things off when you have completed them; make sure you actually follow it and not just tick things off otherwise, we won't take you seriously.
```

Reply:
I might also change it to "otherwise, you won't be taken seriously."